common/decisions/wc_lordaeron_decisions.txt

```diff
@@ -0,0 +1,81 @@
+﻿
+develop_the_undercity = { 
+    ai_check_interval = 6
+	sort_order = 1
+	picture = "gfx/interface/illustrations/decisions/wc_undercity_throne_room_decision.dds"
+	desc = develop_the_undercity_desc
+	selection_tooltip = develop_the_undercity_tooltip
+	major = yes
+
+	is_shown = {
+		NOT = {
+			is_target_in_global_variable_list = {
+				name = unavailable_unique_decisions
+				target = flag:develop_the_undercity
+			}
+		}
```

Comment:
```suggestion
		NOT = {
			title:c_lordaeron = { has_variable = develop_the_undercity }
		}
```
I would suggest to work through this. Global variables must be used only when you dont have any other choice. So fix in other places this thingy

Reply:
ok thanks! i also thought this didn't look right. i will make sure we don't build decisions this way in the future. we will have to figure out how to change the arthas event, as that's where i got this code.

Reply:
We dont need any changes in Arthas events, it has completely different mechanics, so you dont have to compare them. Your events work around one county. So using variable in that county is perfect.